UnitTests/fixtures/LocStructuredOnly.strings

```diff
@@ -0,0 +1,3 @@
+"settings.navigation-bar.title.even.deeper" = "Settings";
+"SETTINGS.USER_PROFILE_SECTION.HEADER_TITLE" = "User Profile Settings";
+"seTTings.uSer-proFile-sectioN.footer_text" = "Here you can change some user profile settings.";
```

Comment:
Thanks for reviewing and helping to improve the PR so quickly. 👍 

Yeah, I'm also interested in what parts of SwiftGen will become obsolete by Xcode 8 – if any. If I remember correctly than Xcode 8 will include some kind of new image and colors within code. Haven't tested though, we're gonna see how that will improve working with resources.